docs_app/src/index.html

```diff
@@ -141,7 +141,7 @@
     <div class="background-sky hero"></div>
     <section id="intro">
       <div class="hero-logo">
-        <img src="assets/images/logos/angular/angular.svg" width="250" height="250">
+        <img src="assets/images/logos/angular/angular.svg" width="250" height="250" alt="RxJS logo">
```

Comment:
This seems to be angular related, none of your business, but a thing we should take care of at some point

Reply:
🤦‍♂️ didn't notice it was the angular logo. I'll change it to `Angular logo`, so if we do address this at some point we don't forget to change the alt attribute.

Reply:
Sorry but you slightly misunderstood me. This part seems to be old stuff of the aio web page we used as basis for the new rxjs docs therefore we should create new content for rxjs in case js is disabled in a browser

Reply:
Yea, this is indeed what I have understood.
I changed the `alt` attribute to not forget it when we change the `noscript` part because I thought we wouldn't want to change this content at this point.

But perhaps you prefer to remove the `alt` attribute? Or to already change the logo to the RxJS logo?

Reply:
Ah ok get it:D I would suggest leave it as it is right now, and we think about a proper noscript content later on!